doc/layers.md

```diff
@@ -0,0 +1,128 @@
+# Layers
```

Comment:
I think this doc could benefit from a quick start guide with "If you want to do "X" use Lock, if you want to do "Y", use Shift, if you want to do "Z", use "Move"

There's some other prose stuff, but I'm happy to tackle that by editing the document.

Reply:
There's a section that is almost your quick start guide... I can move that further up and tweak it a bit to be more like a quick start.